@navikt/aksel-icons/__tests__/validate-svg.test.ts

```diff
@@ -31,27 +32,26 @@ describe(`Each icons has a valid code`, () => {
 
       test(`has valid xml-attr`, () => {
         const root = select(":root", iconAst);
-        const xmlns = root && root.properties ? root.properties.xmlns : null;
+        const xmlns = root?.properties ? root.properties.xmlns : null;
         expect(xmlns).toBe("http://www.w3.org/2000/svg");
```

Comment:
```suggestion
        const xmlns = root?.properties?.xmlns;
        expect(xmlns).toBe("http://www.w3.org/2000/svg");
```
or even
```suggestion
        expect(root?.properties?.xmlns).toBe("http://www.w3.org/2000/svg");
```
Can do the same on the other cases.

Reply:
Updated 🚀